setup.py

```diff
@@ -25,6 +25,12 @@
     author_email="neuropoly@googlegroups.com",
     keywords="",
     packages=find_packages(exclude=["contrib", "docs", "tests"]),
+    entry_points={
+        'console_scripts': [
+            "shim-referencemaps=shimmingtoolbox.cli.referencemaps:main",
+            "shim-b0maps=shimmingtoolbox.cli.b0maps:main"
```

Comment:
Maybe the scripts should just be "referencemaps" and "b0maps"? :shrug: 

I wish `scripts=` worked still.

Reply:
Any reason why `shim-` is added before the function name?

Reply:
we should probably settle on a "shimming-toolbox" prefix. How about `st_`? or do we want something more explicit? like `shimtool_`?
also: separation with _ or - ?

Reply:
I like `st-` better because it is faster but I'm open to other opinions

Reply:
traditionally in the lab we go with snake_case for CLI commands, but i'm happy to reconsider "-" for word separators. it is definitely faster

Reply:
It's just to keep the collection of related tools together. There isn't a namespace system for command line commands, so lots of projects that are made of small related tools do this.

Sticking with snake_case makes the python names and the unix names compatible, so my vote is `st_`.

Reply:
> Sticking with snake_case makes the python names and the unix names compatible, so my vote is st_

out of curiosity, why wouldn't `st-` be python and unix compatible?  is it because "-" could be interpreted as "minus" in python? (it shouldn't because we are not calling CLI names inside code, are we?)

Reply:
Snake_case also works

Reply:
I also like better the "_" by habits. Also a convention for C like programming languages like Python is to use the underscore for exactly the idea of minus symbol since minus operation doesn't need to have spaces before or after the symbol (From what I know)